interface/patient_file/front_payment.php

```diff
@@ -1276,9 +1281,9 @@ function make_insurance() {
                                         $encs[$key]['charges'] += $brow['fee'];
                                         // Add taxes.
                                         $sql_array = array();
-                                        $query = "SELECT taxrates FROM codes WHERE " .
+                                        $query = "SELECT taxrates FROM `codes` WHERE " .
                                         "code_type = ? AND " .
-                                        "code = ? AND ";
+                                        "code = ? AND " .
```

Comment:
hi @juggernautsei , this looks odd here.

Reply:
You are right but the IDE tags it with a red underline (Which the IDE can be wrong). @stephenwaite.
I will revert it back.